docs/includes/excel-custom-functions-manually-register.md

```diff
@@ -1,22 +1,18 @@
 If the `CONTOSO` namespace isn't available in the autocomplete menu, take the following steps to register the add-in in Excel.
 
-### [Excel on Windows or Mac](#tab/excel-windows)
+# [Excel on Windows or Mac](#tab/excel-windows)
 
-1. In Excel, choose the **Insert** tab and then choose the down-arrow located to the right of **My Add-ins**.
+1. In the Excel ribbon, select the **Home** > **Add-ins**.
 
-    :::image type="content" source="../images/select-insert.png" alt-text="Screenshot of the Insert ribbon in Excel on Windows, with the My Add-ins down-arrow highlighted.":::
+1. Under the **Developer Add-ins** section, select **My custom functions add-in** to register it.
 
-1. In the list of available add-ins, find the **Developer Add-ins** section and select **My custom functions add-in** to register it.
-
-    :::image type="content" source="../images/excel-cf-tutorial-register.png" alt-text="Screenshot of the Insert ribbon in Excel on Windows, with the Excel Custom Functions add-in highlighted in the My Add-ins list.":::
+    :::image type="content" source="../images/excel-cf-select-add-in.png" alt-text="The My Add-ins dialog that shows active add-ins, with the My custom function add-in button highlighted.":::
 
 # [Excel on the web](#tab/excel-online)
 
-1. In Excel, choose the **Insert** tab and then choose **Add-ins**.
-
-    :::image type="content" source="../images/excel-cf-online-register-add-in-1.png" alt-text="Screenshot of the Insert ribbon in Excel on the web, with the My Add-ins button highlighted.":::
+1. In Excel, select **Home** > **Add-ins**, then select **Get Add-ins**.
```

Comment:
```suggestion
1. In Excel, select **Home** > **Add-ins**, then select **More Add-ins**.
```

Reply:
![image](https://github.com/OfficeDev/office-js-docs-pr/assets/38896772/b4a3d2d1-45f8-4e6b-9b9f-34b0cec0164d)

Unless I'm mistaken, the "More" button doesn't always show up.

Reply:
I consistently get the "Get Add-ins" option in the WXP desktop clients, and the "More Add-ins" in the web clients.

Reply:
Ah, you're right... What fun!